src/c++/perf_analyzer/concurrency_worker.cc

```diff
@@ -134,23 +139,22 @@ ConcurrencyWorker::CreateContextsAsNecessary()
   // maintain concurrency for this thread.
   size_t active_ctx_cnt = on_sequence_model_ ? thread_config_->concurrency_ : 1;
 
-  if (active_ctx_cnt > ctxs_.size()) {
-    while (active_ctx_cnt > ctxs_.size()) {
-      CreateContext();
+  if (active_ctx_cnt > infer_manager_->GetNumCtxs()) {
```

Comment:
So InferManager basically handles contexts?

Reply:
That's the plan for now, although later I was wondering if the new class should BE the context.  I'll want to sit down with you guys to discuss this class.